kupfer.py

```diff
@@ -19,4 +20,7 @@
 
 if __name__ == '__main__':
     from kupfer import main
+#    from os import chdir
+#    chdir("/home/chaitanya/Documents/PROJECTS/kupfer/")
+#    chdir is required if kupfer script added in OS starup (start on boot). Otherwise all the applications are not indexed.
```

Comment:
I use Arch Linux. May be a bug with that..
But always I have observed that whenever I start Kupfer **after booting**, all the applications are **correctly indexed**. This is not the case when Kupfer is scheduled for autostart.